index.html

```diff
@@ -4,78 +4,99 @@
-    <meta charset="utf-8" name = "description" content="NOAA Fisheries Integrated Toolbox is a portal of modeling, forecasting, and analysis tools for fisheries, protected species, socioeconomics, and ecosystems."/>
+    <link type="text/css" rel="stylesheet" href="./static/simple.css"/>
+    <meta name="google-site-verification" content="U6vzg6_XZ4aLFUQSw7xIysB7E3WtARBQnKU_gbCQWz4"/>
+    <meta charset="utf-8" name="description"
+          content="NOAA Fisheries Integrated Toolbox is a portal of modeling, forecasting, and analysis tools for fisheries, protected species, socioeconomics, and ecosystems."/>
     <meta name="author" content="Corinne Bassin, fisheries.toolbox@noaa.gov">
-    <meta name="keywords" content="NOAA Fisheries Toolbox, NFT, NOAA FIT, NOAA Fisheries Integrated Toolbox, fisheries toolbox, assessment, software, stock, assessment methods, estimation, age based, length based, catch at age, survey, index, graphic interface, NOAA Fisheries, NOAA">
+    <meta name="keywords"
+          content="NOAA Fisheries Toolbox, NFT, NOAA FIT, NOAA Fisheries Integrated Toolbox, fisheries toolbox, assessment, software, stock, assessment methods, estimation, age based, length based, catch at age, survey, index, graphic interface, NOAA Fisheries, NOAA">
 </head>
 <body>
-     <div class="container">
-        <div class="mywrap">
-            <h1 class ="header_title">NOAA Fisheries Integrated Toolbox</h1>
+<div class="container">
+    <div class="mywrap">
+        <h1 class="header_title">NOAA Fisheries Integrated Toolbox</h1>
+    </div>
+    <div class="divider"></div>
+    <div class="mywrap">
+        <div class="box">
+            <div class="title"><a class="title_link" href="/news">NEWS</a></div>
         </div>
-        <div class = "divider"></div>
-     <div class="mywrap">
-        <div class="box"><div class ="title"><a class ="title_link" href="/news">NEWS</a></div></div>
-
-        <div class="box"><div class ="title">TOOLS</div>
-            <ul class ="tools_list">
-                <li style="background-color:rgba(250,250,250,0.7);" ><a href="https://nmfs-fish-tools.github.io"> Fish and Fisheries</a></li>
-                <li style="background-color:rgba(250,250,250,.55);" ><a href="https://noaa-fisheries-integrated-toolbox.github.io/NMFS-Protected-Species-Tools/">Protected Species</a></li>
-                <li style="background-color:rgba(250,250,250,.4);" >Human Dimensions</li>
-                <li style="background-color:rgba(250,250,250,.25);" ><a href="https://nmfs-ecosystem-tools.github.io/">Ecosystems</a></li>
-                <li style="background-color:#0055A4;" ><a href="https://nmfs-general-modeling-tools.github.io">General Modeling Tools</a></li>
+        <div class="box">
+            <div class="title">TOOLS</div>
+            <ul class="tools_list">
+                <li style="background-color:rgba(250,250,250,0.7);">
```

Comment:
Aligning indents and making sentences more well-structured make your codes more readable.